_data/master/navbars/reference.yml

```diff
@@ -132,9 +132,7 @@ toc:
   path: /reference/requirements
 - title: Repo Structure
   path: /reference/repo-structure
-- title: Contribute
-  path: /reference/contribute
-- title: Involved
+- title: Get involved
```

Comment:
should `Involved` be capitalized?

Reply:
OK yes @ozdanborne, I think it's better.